src/Roslyn.Diagnostics.Analyzers/Core/CreateTestAccessor.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Roslyn.Diagnostics.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public class CreateTestAccessor : DiagnosticAnalyzer
+    {
+#pragma warning disable CA1802 // Use literals where appropriate
+        public static readonly string GetTestAccessorMethodName = "GetTestAccessor";
+        public static readonly string TestAccessorTypeName = "TestAccessor";
+#pragma warning restore CA1802 // Use literals where appropriate
+
+        private static readonly LocalizableString s_localizableTitle = new LocalizableResourceString(nameof(RoslynDiagnosticsAnalyzersResources.CreateTestAccessorTitle), RoslynDiagnosticsAnalyzersResources.ResourceManager, typeof(RoslynDiagnosticsAnalyzersResources));
+        private static readonly LocalizableString s_localizableMessage = new LocalizableResourceString(nameof(RoslynDiagnosticsAnalyzersResources.CreateTestAccessorMessage), RoslynDiagnosticsAnalyzersResources.ResourceManager, typeof(RoslynDiagnosticsAnalyzersResources));
+        private static readonly LocalizableString s_localizableDescription = new LocalizableResourceString(nameof(RoslynDiagnosticsAnalyzersResources.CreateTestAccessorDescription), RoslynDiagnosticsAnalyzersResources.ResourceManager, typeof(RoslynDiagnosticsAnalyzersResources));
+
+        internal static readonly DiagnosticDescriptor CreateTestAccessorRule = new DiagnosticDescriptor(
```

Comment:
Are you not supposed to use the helper instead of the `new DiagnosticDescriptor`?

Reply:
Which helper is it? This was originally written back in March 2019 so the helper didn't exist.

Reply:
The new helper isn't used in this project.

Reply:
Yes, the helper is just for CA rules and is now enforced in build in the relevant project(s).